packaging/docker/run.sh

```diff
@@ -46,6 +46,16 @@ if [ -n "${PGID}" ]; then
   usermod -a -G "${PGID}" "${DOCKER_USR}" || echo >&2 "Could not add netdata user to group docker with ID ${PGID}"
 fi
 
+# Needed to read Proxmox VMs and (LXC) containers configuration files (name resolution + CPU and memory limits)
+HOST_WWW_DATA_GID=$(stat -c %g /host/etc/pve 2>/dev/null || true)
+
+if [ -n "${HOST_WWW_DATA_GID}" ]; then
+  echo "Creating host-www-data group ${HOST_WWW_DATA_GID}"
+  addgroup -g "${HOST_WWW_DATA_GID}" "host-www-data" || echo >&2 "Could not add group with ID ${HOST_WWW_DATA_GID}, its already there probably"
```

Comment:
Have we confirmed that this works even if a group with the same GID exists in the container already? If not we need to check it, because there will almost certainly be an overlap with the Debian base images.

Reply:
Yes, this is actually my case. 33 is `www-data` on the host, and 33 is `xfs` inside the container. We just need to add netdata to 33 (`xfs` or whatever doesn't matter).